src/middlewares/signUpValidation.js

```diff
@@ -0,0 +1,33 @@
+// eslint-disable-next-line import/no-unresolved
+import Joi from 'joi';
+
+export default (req, res, next) => {
+  const signUpSchema = {
+    firstName: (Joi.string().required()).replace(/\s/g, '').error(() => ({
```

Comment:
The replace method won't work here because it's not called on the actual value but on Joi object